api/src/pcapi/core/finance/api.py

```diff
@@ -811,7 +811,9 @@ def edit_business_unit(business_unit: models.BusinessUnit, siret: str) -> None:
         raise ValueError("Cannot edit a business unit that already has a SIRET.")
 
     validation.check_business_unit_siret(business_unit, siret)
-
+    venue = offerers_models.Venue.query.filter(offerers_models.Venue.siret == siret).one()
+    if not venue is None:
+        business_unit.name = venue.publicName
```

Comment:
`venue` ne peut pas être `None` ici, car `one()` renvoie un élément s'il existe, ou lève une exception (s'il n'y a pas d'éléments ou plus d'un).

(Au passage, je pense qu'on ne devrait pas gérer ce cas (pas de lieu avec le SIRET d'une BU). Ça ne devrait pas arriver, et on devrait échouer bruyamment.)

```suggestion
    business_unit.name = venue.publicName
```